src/services/mapboxService.tsx

```diff
@@ -1,4 +1,5 @@
 import mapboxgl from 'mapbox-gl'
+import { MapProjection } from '../components/organisms/MapBoxMap/MapboxMap'
```

Comment:
this import is wrong, should be import { MapProjection } from '../components/atoms/ToggleMapProjectionButton/ToggleMapProjection'